release-notes/release-notes59.md

```diff
@@ -219,6 +219,22 @@ The {{site.productname}} 5.9 release includes an accompanying release of the **P
 
 For information on the PowerPaste plugin, see: [PowerPaste plugin]({{site.baseurl}}/plugins/premium/powerpaste/).
 
+### Real-Time Collaboration 1.0.0
```

Comment:
This will actually be 1.0.1 if you're worried :shrug:

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/2091/commits/b3091a1ad3845121811a20d729af9663174b26ec